solution_runner/commands/setup_command.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime
+from pathlib import Path
+
+import click
+import requests
+
+from . import consts
+from .consts import Directories, FileExtensions
```

Comment:
you imported consts one line before, why did you import specific classes from that same consts file?

Reply:
Because `consts.Directories.SOLUTIONS` is too long. This is usually the style I go with.

Reply:
I think that the better way is to import specific things from a page and not using import consts at all

Reply:
I prefer to not do imports that way, it's not clear where the object came from that way.